### PR TITLE
ref(envelope_manager): Remove from_registry calls from the service

### PR DESCRIPTION
**Pull request by olksdr**
This PR removes the `from_registry` calls from the `EnvelopeMangerService`.
Removed un-used code.

#skip-changelog

ref: #1818

### CODE REVIEW

File: relay-server/src/actors/envelopes.rs [hunk 1]
```diff
@@ -201,15 +198,29 @@ impl FromMessage<SendMetrics> for EnvelopeManager {
 #[derive(Debug)]
 pub struct EnvelopeManagerService {
     config: Arc<Config>,
+    aggregator: Addr<Aggregator>,
+    enveloper_processor: Addr<EnvelopeProcessor>,
+    test_store: Addr<TestStore>,
+    upstream_relay: Addr<UpstreamRelay>,
```
Comment (jjbayer) [inline relay-server/src/actors/envelopes.rs:204]: nit: Should we make this an "address book" again?

File: relay-server/src/service.rs [hunk 2]
```diff
@@ -120,20 +111,7 @@ impl ServiceState {
...
-            let rt = create_runtime("store-rt", 1);
-            let store = StoreService::create(config.clone())?.start_in(&rt);
-            envelope_manager.set_store_forwarder(store);
-            _store_runtime = Some(rt);
-        }
-
-        let envelope_manager = envelope_manager.start();
-        let test_store = TestStoreService::new(config.clone()).start();
-
+        let (envelope_manager, envelope_manager_rx) = channel(EnvelopeProcessorService::name());
```
Comment (jjbayer) [inline relay-server/src/service.rs]: I assume this is to resolve the circular dependency between manager and processor? Could you add a comment explaining this?

**Review by jjbayer — APPROVED** [review]

File: relay-server/src/actors/envelopes.rs (hunk 1 above)
Comment (olksdr) [reply relay-server/src/actors/envelopes.rs:204]: This for me still fine and I would keep it separated for now and I would like to keep this still as it is.

File: relay-server/src/service.rs (hunk 2 above)
Comment (olksdr) [reply relay-server/src/service.rs]: Yeah, we have to get an address of the manager before we can use it in the processor, still we also have to have a processor address in the manager here.

Comment (olksdr) [reply relay-server/src/service.rs]: done in [f0de37a](https://github.com/getsentry/relay/pull/1956/commits/f0de37a6ec95588d53b41fa020fd952b7b9562be)

**Pull request merged**